### PR TITLE
[ci] Support to merge SONiC package versions change pr automatically

### PR DESCRIPTION
**Pull request by xumia**
#### Why I did it
Support to merge the SONiC package versions upgrade pull request automatically.

#### How I did it
Add the automerge tag in the pull request.
1. Sent by the author mssonicbld
2. Only contains changes in files/build/**/*
3. Only for the specified branch 202012

Merge the pull request.
1. Sent by the author mssonicbld
2. The Azure Pipelines checker complete and successful
3. The pull request contains label automerge

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012

### CODE REVIEW

File: .github/workflows/automerge.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#
+
+name: automerge
+on:
+  check_suite:
+    types:
+      - completed
+jobs:
+  automerge:
+    if: github.event.check_suite.app.name == 'Azure Pipelines' &&  github.event.check_suite.conclusion == 'success'
```
Comment (qiluo-msft) [inline .github/workflows/automerge.yml:11]: >  [](start = 65, length = 1)

one extra blank

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+automerge:
+- all: ['files/build/**/*']
```
Comment (qiluo-msft) [inline .github/labeler.yml:2]: >files/build/**/* [](start = 9, length = 16)

Do you mean `files/build/versions/**` ?

Comment (xumia) [reply .github/labeler.yml:2]: Yes, but not supported.
I want to add the label when only change in files/build/versions/**/* or files/build/versions/*, and not other change.
See https://github.com/actions/labeler.

all: match ALL globs against ALL changed paths

File: .github/workflows/automerge.yml (hunk 1 above)
Comment (xumia) [reply .github/workflows/automerge.yml:12]: Fixed.

Comment (qiluo-msft) [reply .github/workflows/automerge.yml:12]: Sorry, I mean you have one extra blank after `&&`

---
In reply to: [607427677](https://github.com/Azure/sonic-buildimage/pull/7065#discussion_r607427677) [](ancestors = 607427677)

File: .github/labeler.yml (hunk 2 above)
Comment (xumia) [reply .github/labeler.yml]: Qi, it works as expected, fixed, thanks.

**Review by qiluo-msft — APPROVED** [review]

**Review by lguohan — APPROVED** [review]

**Pull request merged**